docs/_content/standaard/standaard.md

```diff
@@ -148,6 +148,10 @@ Componenten dienen events te publiceren naar (een)
 notificatierouteringcomponent(en) (NRC). De NRC MOET volledig de
 [`openapi.yaml`](../../../api-specificatie/nrc/openapi.yaml) implementeren.
 
+Consumers van events dienen de consumer
```

Comment:
Voorstel:

Applicaties MOGEN een abonnement nemen op 1 of meer kanalen. Deze applicaties zijn dan event consumers. Een event consumer MOET de [API volgens `openapi.yaml`](../../../api-specificatie/nrc/consumer-api/openapi.yaml) implementeren om berichten te kunnen ontvangen.

Componenten geven aan indien het nemen van een abonnement op bepaalde kanalen verplicht is. Deze componenten zijn dan naast provider ook een event consumer.

Reply:
Opgenomen :+1: